rust-runtime/aws-smithy-types/Cargo.toml

```diff
@@ -13,9 +13,10 @@ repository = "https://github.com/smithy-lang/smithy-rs"
 [features]
 byte-stream-poll-next = []
 http-body-0-4-x = ["dep:http-body-0-4"]
+http-body-1-x = ["dep:http-body-1-0", "dep:http-body-util", "http-body-0-4-x"]
 hyper-0-14-x = ["dep:hyper-0-14"]
 rt-tokio = [
-    "dep:http-body-0-4",
+    "http-body-0-4-x",
```

Comment:
This will permanently tie rt-tokio to the http-body-0-4-x feature since it brings in the API for 0.4.